ip/mp4.c

```diff
@@ -171,11 +171,13 @@ static int mp4_open(struct input_plugin_data *ip_data)
 	NeAACDecSetConfiguration(priv->decoder, neaac_cfg);
 
 	/* open mpeg-4 file */
-#ifdef MP4_DETAILS_ALL
-	priv->mp4.handle = MP4Read(ip_data->filename, 0);
-#else
+#ifdef __APPLE__
```

Comment:
The API changed with version 2.1.0 so the conditional should be
```suggestion
#if MP4V2_PROJECT_version_major < 2 ||  (MP4V2_PROJECT_version_major == 2 && MP4V2_PROJECT_version_minor < 1)
```

Reply:
Good Idea.